specification/cognitiveservices/data-plane/Personalizer/preview/v1.0/Personalizer.json

```diff
@@ -0,0 +1,1042 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v1.0",
+    "title": "Personalizer Client",
+    "description": "Personalizer Service is an Azure Cognitive Service that makes it easy to target content and experiences without complex pre-analysis or cleanup of past data. Given a context and featurized content, the Personalizer Service returns your content in a ranked list. As rewards are sent in response to the ranked list, the reinforcement learning algorithm will improve the model and improve performance of future rank calls."
+  },
+  "basePath": "/personalizer/v1.0",
+  "paths": {
+    "/configurations/service": {
+      "get": {
+        "tags": [
+          "Configuration"
+        ],
+        "summary": "Get the service configuration.",
+        "operationId": "GetServiceConfiguration",
```

Comment:
OperationIds must be of the form `Noun_Verb`
In this case it should be `ServiceConfiguration_Get`
Applies to all the operation ids

Reply:
fixed